fooof/objs/fit.py

```diff
@@ -723,6 +738,24 @@ def set_debug_mode(self, debug):
         self._debug = debug
 
 
+    def set_check_modes(self, check_freqs=None, check_data=None):
+        """Set check modes, which controls if an error is raised based on check on the inputs.
+
+        Parameters
+        ----------
+        check_freqs : bool, optional
+            Whether to run in check freqs mode, which checks the frequency data.
+        check_data : bool, optional
```

Comment:
if these are bool, maybe they be defaulted to false in the signature?

Reply:
The reason I went with None is so one can do something like `fm.set_check_modes(check_data=False)`, without specifying anything for `check_freqs` such that `check_freqs` won't be changed - if the default value was False, this would set `_check_freqs` to False, which might not be intended. Does that make sense?

Reply:
Oh I see. That makes sense. I guess it would work if the default was true, so it matched the private attribute set in the init? There may be other moving pieces here I'm missing though.